lib/message_format.dart

```diff
@@ -808,12 +808,12 @@ class _TypeAndVal<T, V> {
 enum _ElementType { string, block }
 
 class _ElementTypeAndVal extends _TypeAndVal<_ElementType, String> {
-  _ElementTypeAndVal(var type, var value) : super(type, value);
+  _ElementTypeAndVal(_ElementType type, String value) : super(type, value);
 }
 
 /// Block type.
 enum _BlockType { plural, ordinal, select, simple, string, unknown }
 
 class _BlockTypeAndVal extends _TypeAndVal<_BlockType, Object> {
-  _BlockTypeAndVal(var type, var value) : super(type, value);
+  _BlockTypeAndVal(_BlockType type, Object value) : super(type, value);up 
```

Comment:
@lrhn - was this trailing `up` intentional?

Reply:
Absolutely not!